### PR TITLE
Tkakar/cat 613 add status icon multiassay

### PR DESCRIPTION
**Pull request by tkakar**
This PR merges the addition of status icons to the details page for multi-assay dataset relationships

### CODE REVIEW

File: context/app/static/js/components/detailPage/multi-assay/RelatedMultiAssayLinks/style.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
```
Comment (NickAkhmetov) [inline context/app/static/js/components/detailPage/multi-assay/RelatedMultiAssayLinks/style.ts:1]:
```suggestion
import { styled } from '@mui/material/styles';
```

Importing the styled utility from MUI will make it more straightforward for us to eventually upgrade to a more performant styling approach (likely https://github.com/mui/pigment-css once it's mainlined in MUI 6). It also makes it easier to access theme values in the styles via TypeScript hints.

File: context/app/static/js/components/detailPage/multi-assay/RelatedMultiAssayLinks/RelatedMultiAssayLinks.tsx [hunk 2]
```diff
@@ -37,17 +39,22 @@ type Entries<T> = {
...
   return (
-    <Typography>
-      {assay_display_name}: <InternalLink href={`/browse/dataset/${uuid}`}>{hubmap_id}</InternalLink>
-    </Typography>
+    <SecondaryBackgroundTooltip title={status}>
+      <Typography>
+        <FlexContainer>
+          {assay_display_name}:<InternalLink href={`/browse/dataset/${uuid}`}>{hubmap_id}</InternalLink>
+          <StatusIcon status={status} />
+        </FlexContainer>
```
Comment (NickAkhmetov) [inline context/app/static/js/components/detailPage/multi-assay/RelatedMultiAssayLinks/RelatedMultiAssayLinks.tsx:49]:
```suggestion
        <Stack direction="row" useFlexGap gap={0.5} alignItems="center">
          {assay_display_name}:<InternalLink href={`/browse/dataset/${uuid}`}>{hubmap_id}</InternalLink>
          <StatusIcon status={status} />
        </Stack>
```

The [MUI `Stack` component](https://mui.com/material-ui/react-stack/) is very useful in cases that call for a flex container on the go 😄

File: context/app/static/js/components/detailPage/multi-assay/RelatedMultiAssayLinks/style.ts
Comment (NickAkhmetov) [inline context/app/static/js/components/detailPage/multi-assay/RelatedMultiAssayLinks/style.ts:1]: With the suggested change to `Stack` above, we should be able to remove this file.

**Review by NickAkhmetov — CHANGES_REQUESTED** [review]
Great work! Just a minor suggestion re: using the `Stack` utility instead of introducing a single-use styled component.

File: CHANGELOG-add-status-icon-to-multiassay-listing [hunk 3]
```diff
@@ -0,0 +1 @@
+- Added status icon to the processed datasets to differntiate between different runs
```
Comment (john-conroy) [inline CHANGELOG-add-status-icon-to-multiassay-listing:1]: This file is missing the `.md` extension.

File: CHANGELOG-fix-changelog-entry.md [hunk 4]
```diff
@@ -0,0 +1 @@
+- Fix changelog formatting for v0.101.1 entries.
```
Comment (john-conroy) [inline CHANGELOG-fix-changelog-entry.md:1]: This changelog has already been compiled and needs to be removed - sorry this is a pain.

File: CHANGELOG-add-status-icon-to-multiassay-listing.md [hunk 5]
```diff
@@ -0,0 +1 @@
+- Added status icon to the processed datasets to differntiate between different runs
```
Comment (NickAkhmetov) [inline CHANGELOG-add-status-icon-to-multiassay-listing.md:1]:
```suggestion
- Added status icon to the processed datasets to differentiate between different runs.
```

Minor spelling fix/added comma at the end - if you can re-request review after fixing this, I'll sign off on it 😄

**Review by NickAkhmetov — APPROVED** [review]
🚀

**Pull request merged**